esmvalcore/_main.py

```diff
@@ -436,11 +432,19 @@ def display(lines, out):
             logging.basicConfig()
         logger.exception(
             "Program terminated abnormally, see stack trace "
-            "below for more information",
+            "below for more information:",
             exc_info=True)
         logger.info(
-            "If you suspect this is a bug or need help, please open an issue "
-            "on https://github.com/ESMValGroup/ESMValTool/issues and attach "
-            "the run/recipe_*.yml and run/main_log_debug.txt files from the "
-            "output directory.")
+            "\n"
+            "If you have a question or need help, please start a new "
+            "discussion on "
+            "https://github.com/ESMValGroup/ESMValTool/discussions?"
+            "discussions_q=category%3AQ%26A"
```

Comment:
```suggestion
            "https://github.com/ESMValGroup/ESMValTool/discussions"
```

Reply:
I suggest we point to the top level url for discussions for now since I'm not sure if we will be making any tweaks to the setup and/or github may change it.

Reply:
I thought it might be nice to use a single place for questions for help, but indeed if you think that they might change the available labels that's not very useful now.

Reply:
Removed the category